src/app/lockers/page.tsx

```diff
@@ -0,0 +1,34 @@
+import LockerIdForm from './LockerIdForm'
+import LockerList from './LockerList'
+import QRButton from './QRButton'
+import PageWrapper from '@/components/PageWrapper/PageWrapper'
+import LockerPagingProvider from '@/context/paging/LockerPaging'
+import { getUser } from '@/auth/getUser'
+
+export default async function Lockers() {
+    await getUser({
+        userRequired: true,
```

Comment:
Why is user requiered

Reply:
I was thinking that only people who have an user should be able to see the reservations of others

Reply:
In production: LOCKER_READ will obviously not be a standard permission. But in theory there is no reason to require a user.